src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -104,12 +104,18 @@ object GermlineAssemblyCaller {
       val referenceKmerSink = currentReference.takeRight(kmerSize)
 
       // If the current window size doesn't cover the kmer size we won't be able to find the reference start/end
-      if (currentReference.size < kmerSize)
+      if (currentReference.size < kmerSize || referenceKmerSource == referenceKmerSink)
```

Comment:
hm, I'm not sold on this added clause. is there nothing useful we can do in regions where the reference as a k-repeat? I can imagine a reasonable-sized window with identical `k`-mers on each end where we'd still want to call variants inside it..